### PR TITLE
Feature/readme docs

### PR DESCRIPTION
**Pull request by axl1313**
Update readme docs w/ python API and updated CLI information

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,330 +1,104 @@
 # cleanlab-studio [![Build Status](https://github.com/cleanlab/cleanlab-studio/workflows/CI/badge.svg)](https://github.com/cleanlab/cleanlab-studio/actions?query=workflow%3ACI) [![PyPI](https://img.shields.io/pypi/v/cleanlab-studio.svg)][PyPI]
 
-Command line interface for all things [Cleanlab Studio](https://cleanlab.ai/studio/). Upload datasets and download cleansets (cleaned datasets) from Cleanlab Studio in a single line of code!
+Command line and Python library interface to [Cleanlab Studio](https://cleanlab.ai/studio/). Upload datasets and download cleansets (cleaned datasets) from Cleanlab Studio in a single line of code!
 
 - [Installation](#installation)
 - [Quickstart](#quickstart)
 - [Reference](#reference)
```
Comment (ryansingman) [inline README.md:7]: broken link for rn -- let's put the schema section under a section called `Advanced Usage`, and link to that here

File: README.md [hunk 2]
```diff
@@ -1,330 +1,104 @@
...
-
-To download clean labels (i.e., labels that have been fixed through the Cleanlab Studio interface), first find the ID of the cleanset. You can find this by clicking the "Export Cleanset" button in the top right of a project page.
-
-```bash
-cleanlab cleanset download --id [cleanset ID]
-```
-
-The above command only downloads corrected labels. You can also download corrected labels and combine them with your local dataset in a single command:
+# create your Cleanlab Studio API client with your API key, found here: https://app.cleanlab.ai/account
+studio = Studio(<your API key>)
```
Comment (ryansingman) [inline README.md:32]: remove caps from API -- weird syntax highlighting

File: README.md [hunk 3]
```diff
@@ -1,330 +1,104 @@
...
-    "vizzy_id": "4",
-    "vizzy_path": "Dataset/mocha/mocha.jpeg",
-    "label": "dog"
-  }
-]
+# download your cleanset or apply corrections to your local Pandas or PySpark dataset!
+# @ANISH !!! how do we want to tell users to find their cleanset ID?
+# I think we should either: Add the cleanset ID to the version history page or display it somewhere obvious on the project page
+cleanset = studio.download_cleanlab_columns(<your cleanset ID>)
+corrected_dataset = studio.apply_corrections(<your_dataset>, <your cleanset ID>)
```
Comment (ryansingman) [inline README.md:43]: remove caps from ID -- weird syntax highlighting

File: README.md [hunk 4]
```diff
@@ -1,330 +1,104 @@
...
+  - PySpark DataFrame *(Python library only)*
+  - more to come!
+- Image
+  - CSV (external media)
+  - JSON (external media)
+  - XLS/XLSX (external media)
+  - Pandas DataFrame (external media) *(Python library only)*
+  - PySpark DataFrame (external media) *(Python library only)*
+  - Simple ZIP upload
+  - Metadata ZIP upload
```
Comment (ryansingman) [inline README.md:69]: add more to come here

**Review by ryansingman — CHANGES_REQUESTED** [review]

File: README.md [hunk 5]
```diff
@@ -1,330 +1,107 @@
...
 pip install --upgrade cleanlab-studio
 ```
 
 ## Quickstart
 
-1. If this is your first time using the Cleanlab CLI, authenticate with `cleanlab login`. You can find your API key at <https://app.cleanlab.ai/upload>, by clicking on "Upload via Command Line".
+### Python API
 
-2. Upload your dataset ([image](#upload-an-image-dataset), [text](#upload-a-text-dataset), or [tabular](#upload-a-tabular-dataset)) using `cleanlab dataset upload`.
+You can find your API key at
```
Comment (anishathalye) [inline README.md:27]: missing the rest of the sentence

File: README.md [hunk 6]
```diff
@@ -1,330 +1,107 @@
...
-    "label": "dog"
-  },
-  {
-    "vizzy_id": "4",
-    "vizzy_path": "Dataset/mocha/mocha.jpeg",
-    "label": "dog"
-  }
-]
+# download your cleanset or apply corrections to your local Pandas or PySpark dataset!
+# @ANISH !!! how do we want to tell users to find their cleanset ID?
```
Comment (anishathalye) [inline README.md:40]: Why not point people to the export cleanset button? (And in particular, the export via CLI tab?)

If that tab needs to be improved, we can improve the UI/text there.

File: README.md [hunk 7]
```diff
@@ -1,330 +1,107 @@
...
+    ```
+    {
+      <name_of_column_to_override>: {
+        "data_type": <desired_data_type>,
+        "feature_type": <desired_feature_type>,
+      },
+      ...
+    }
+    ```
+    Include entries for all the columns you'd like to override
```
Comment (anishathalye) [inline README.md:92]: was this meant to be indented like this?

**Pull request merged**